src/MudBlazor/Enums/InputType.cs

```diff
@@ -19,6 +19,22 @@ public enum InputType
         [Description("tel")]
         Telephone,
         [Description("url")]
-        Url
+        Url,
+        [Description("color")]
+        Color,
+        [Description("date")]
+        Date,
+        [Description("datetime-local")]
+        DateTimeLocal,
+        [Description("month")]
+        Month,
+        [Description("time")]
+        Time,
+        [Description("week")]
+        Week,
+        [Description("tel")]
+        Tel
```

Comment:
```suggestion
        Week
```
There's already a telephone type defined